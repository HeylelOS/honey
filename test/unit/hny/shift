#!/bin/sh

set -e

"${TEST_BIN}" shift 'archive' 'archive-1.0.0'
"${TEST_BIN}" shift 'arxiv' 'archive'

[ -h "${HNY_PREFIX}/archive" ]
[ -h "${HNY_PREFIX}/arxiv" ]

basename -- `readlink -- "${HNY_PREFIX}/archive"`
basename -- `readlink -- "${HNY_PREFIX}/arxiv"`


