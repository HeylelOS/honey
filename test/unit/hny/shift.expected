archive-1.0.0
archive
